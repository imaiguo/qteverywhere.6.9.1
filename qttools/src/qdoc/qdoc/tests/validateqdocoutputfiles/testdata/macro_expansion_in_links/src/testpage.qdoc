// Copyright (C) 2025 The Qt Company Ltd.
// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GFDL-1.3-no-invariants-only

/*!
    \page testpage.html
    \title Test Page

    This is a page for testing purposes.
*/

