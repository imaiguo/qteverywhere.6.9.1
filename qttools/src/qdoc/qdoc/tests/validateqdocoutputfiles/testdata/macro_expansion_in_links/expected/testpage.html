<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
<!-- testpage.qdoc -->
  <title>Test Page | Qt</title>
</head>
<body>
<div class="sidebar"><div class="sidebar-content" id="sidebar-content"></div></div>
<h1 class="title">Test Page</h1>
<!-- $$$testpage.html-description -->
<div class="descr" id="details">
<p>This is a page for testing purposes.</p>
</div>
<!-- @@@testpage.html -->
</body>
</html>
